transformations/discourse_marker_substitution/transformation.py

```diff
@@ -0,0 +1,141 @@
+import random
+import re
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+"""
+Substitution of discourse markers with semantically equivalent marker
+"""
+
+CLASS_TO_MARKERS = {
+    "Comparison.Contrast": ["but", "however", "on the contrary"],
+    "Comparison.Contrast.Juxtaposition": ["by comparison", "whereas"],
+    "Contingency.Cause.Reason": ["because", "as", "since", "inasmuch as"],
+    "Contingency.Cause.Result": [
+        "so",
+        "as a result",
+        "consequently",
+        "thus",
+        "therefore",
+        "accordingly",
+        "hence",
+    ],
+    "Expansion.Alternative.Chosen alternative": ["instead"],
+    "Expansion.Alternative.Conjunctive": ["or"],
+    "Expansion.Conjunction": [
+        "and",
+        "in fact",
+        "while",
+        "also",
+        "furthermore",
+        "in addition",
+        "meanwhile",
+        "moreover",
+        "similarly",
+        "further",
+        "in turn",
+        "likewise",
+        "besides",
+        "additionally",
+    ],
+    "Expansion.Instantiation": ["for example", "for instance"],
+    "Expansion.Restatement.Equivalence": ["in other words"],
+    "Expansion.Restatement.Generalization": ["in sum"],
+    "Expansion.Restatement.Specification": ["specifically", "in particular"],
+    "Temporal.Asynchronous.Precedence": ["then", "subsequently", "finally", "later"],
+    "Temporal.Asynchronous.Succession": ["previously", "earlier", "after"],
+    "Temporal.Synchrony": ["at the time", "when", "at the same time"],
+}
+
+MARKER_TO_CLASS = {
```

Comment:
Could you not create this one automatically from the information from `CLASS_TO_MARKER` to improve maintenance?